static/links.less

```diff
@@ -1,10 +1,19 @@
 @import "ui-variables";
 
 a {
-  cursor: pointer;
   color: @text-color-highlight;
   &:hover {
     color: @text-color-highlight;
     text-decoration: underline;
   }
+
+  &[href^="http"],
```

Comment:
Found it - [grunt-lesslint](https://github.com/jgable/grunt-lesslint) actually uses [csslint](https://github.com/CSSLint/csslint) for linting once the Less is compiled, and csslint wasn't configured properly for the lesslint task (it was complaining about regex-like selectors being slow).

@kevinsawicki Is there any reason why we have both grunt-csslint and grunt-lesslint set up? As far as I can tell they seem a bit redundant, since grunt-lesslint is basically just csslint + sourcemaps to source Less files.